cli/README.md

```diff
@@ -55,7 +55,7 @@ Subcommands:
 
 Options:
   -d, --database  # Preconfigure for selected database. Options: pg | mysql | sqlite (default: pg)
-  -m, --model     # Preconfigure for selected model. Options: granite | crecto (default: granite)
+  -m, --model     # Preconfigure for selected model. Options: granite (default: granite)
```

Comment:
The MR I proposed to remove creto also removes the -m option, so if it get approved, this entire line need to be removed, not modified.

Reply:
@hugopl I disagree. In the future more ORMs may be added we don't need to remove the option to select an ORM entirely.

Reply:
There's a discussion related to this here: https://github.com/amberframework/amber/issues/942

I like the idea of supporting just one ORM to make things easier, and since granite is under amber umbrella, seems natural it to be the chosen one. But I'm waiting for some decision form core devs before do any change in the PR.